core/src/main/resources/css/atoms/_audio.scss

```diff
@@ -181,15 +181,15 @@ input[type=range]:focus {
 
 .atom--audio__progress-bar input[type=range]::-moz-range-progress {
   .garnett--pillar-news      &,
-  .content--pillar-news      & { background-color: linear-gradient(to right, $c-pillar--news 0%, #afafaf 0%); }
+  .content--pillar-news      & { background-image: linear-gradient(to right, $c-pillar--news 100%, #afafaf); }
```

Comment:
How could I miss that 😵 Sorry.

Reply:
No worries, I _wrote_ it and didn't see it! 🙄